src/foam/util/NumberParser.js

```diff
@@ -0,0 +1,341 @@
+/**
+ * @license
+ * Copyright 2025 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.util',
```

Comment:
Thinking, package foam/lib/parse (or foam/parse) is more suitable for additional parsers (and tests).

Reply:
done.